pr_agent/tools/pr_update_changelog.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, pr_url: str, cli_mode=False, args=None):
                                           get_settings().pr_update_changelog_prompt.user)
 
     async def run(self):
-        assert type(self.git_provider) == GithubProvider, "Currently only Github is supported"
+        # assert type(self.git_provider) == GithubProvider, "Currently only Github is supported"
```

Comment:
**Suggestion:** Consider replacing the assertion with a more user-friendly error handling mechanism. Assertions can be turned off globally in the Python interpreter, and they are typically used for debugging, not for handling run-time errors.
```suggestion
        if not isinstance(self.git_provider, GithubProvider):
            raise NotImplementedError("Currently only Github is supported")
```